sport/app/football/model/model.scala

```diff
@@ -164,6 +164,17 @@ object CompetitionDisplayHelpers {
       .replace("Holland", "The Netherlands")
   }
 
+  // This function has specifically raised for the 2021 Euros after some clarification.
+  // "We're happy with "Netherlands" as a label, like the keyword for football and also the news one you can see here:
+  // https://www.theguardian.com/world/2021/jun/11/princess-amalia-heir-to-dutch-throne-waives-right-to-yearly-income"
+  // - Philip Cornwall, 16/6/21
+  // I don't know if this is permanent going forward hence the separate function.
+  def cleanTeamName2021(teamName: String): String = {
```

Comment:
Ha this feels pretty unrelated to this PR but I guess we can keep it now.

Reply:
I think there's a few commits / PRs in here which aren't in Pascals branch which is why we're seeing the extra PR's, if it was updated against main I'd expect them to disapear